test/loop/Makefile

```
@@ -2,3 +2,8 @@ CC ?= gcc
 
 loop: loop.c
 	$(CC) $^ -o $@
+
+clean:
+	rm -f loop
```

I think we would want to silence the command itself, like in the other Makefiles
```suggestion
	@rm -f loop
```